src/Persistence.LinnApps/Repositories/StockLocatorRepository.cs

```diff
@@ -35,6 +35,14 @@ public IQueryable<StockLocator> FilterBy(Expression<Func<StockLocator, bool>> ex
                 .Include(l => l.Part);
         }
 
+        public IQueryable<StockLocator> FilterByWildcard(string search)
```

Comment:
If this works well/is quick, do you think it'd be worth us just making the .Where EF.Functions.Like below our sort of go to implementation of the normal FilterBy, or do you think it's worth keeping it separate? It'll still work fine without a wildcard, won't it?

Reply:
It is fast, much faster than the query EF evaluates (often clientside in memory) if you pass some regex to achieve 'LIKE' in our FilterBy(...). 

I wouldn't advocate for repalcing FilterBy though. It's nice and open because you can pass in an expression from the outside world that can do anything you like. In theory you could pass an expression with EF.Functions.Like but that would require your calling context to have EntityFramework installed. Often we're calling repository methods from our Domain which certainly doesn't want EntityFramework or whatever as a depdendency. 

I think the solution is to have both in tandem. Maybe write an<code>IQueryable<T> FilterWithWildcard(string searchExpressionWithWildcards)</code>  into our IRepositories.

Reply:
Nice! 

Yeah sorry I had a moment and forgot we pass any expression in to the repository filterBy methods and not just a string, both would be good!

Reply:
no reason not to have both